neurom/apps/cli.py

```diff
@@ -76,6 +76,10 @@ def view(input_file, is_3d, plane, backend, realistic_diameters):
 
     plot(load_morphology(input_file))
     if is_matplotlib:
+        try:
+            plt.axis('equal')
+        except NotImplementedError:  # for python 3.7 it will not have equal scales
+            plt.axis('auto')
```

Comment:
Isn't `auto` the default behavior? If it is you can pass py37 case.

Reply:
no it's not, auto makes the plot with entire neuron, but not equal x/y scales, something weird is happening in the use of matplotlib, but in a script, I can always add this 'equal' line after a plot

Reply:
Ah ok, I see.